app/src/main/java/kr/ac/kaist/team888/core/Stroke.java

```diff
@@ -178,13 +182,13 @@ public StrokeBuilder setEndPoint(Point2D endPoint) {
       return this;
     }
 
-    public StrokeBuilder setControlPoint(Point2D controlPoint) {
-      this.controlPoint = controlPoint;
+    public StrokeBuilder addControlPoint(Point2D controlPoint) {
```

Comment:
Building Stroke from builder is continuous act. If we make setter rather than adder, we need to initialize control points before we start to build. Give me comment after see below examples.

```java
Stroke stroke = new Stroke.StrokeBuilder()
  .setStartPoint(new Point2D(...))
  .setControlPoints(new ArrayList<Point2D> () {
    new Point2D(...), new Point2D(...))
  .setEndPoint(new Point2D(...))
  .build();
```

```java
Stroke stroke = new Stroke.StrokeBuilder()
  .setStartPoint(new Point2D(...))
  .addControlPoint(new Point2D(...))
  .addControlPoint(new Point2D(...))
  .setEndPoint(new Point2D(...))
  .build();
```